tools/autograd/load_derivatives.py

```diff
@@ -343,6 +360,10 @@ def set_up_derivatives(f: NativeFunction) -> Tuple[
 
         all_arg_names = [a.name for a in cpp_arguments(f)]
 
+        available_named_gradients = [f'grad_{ret.name}'
```

Comment:
I think I *must*.

If I don't, this will be wrong, no?

if I have `name: func() -> (Tensor x, bool y, Tensor z)`, `grads` will be `{x, z}`, right?

I don't see any functions that return multiple outputs where one is not a `Tensor` (some do return `Tensor[]`) though. Let me test this out of curiosity.

Reply:
I added a test for this but I haven't implemented it yet.

Reply:
implemented just by checking if the return type `is_tensor_like()` but this seems insufficient.

Reply:
this is addressed (see new PR).